packages/twenty-server/src/core/billing/controllers/product-price.controller.ts

```diff
@@ -0,0 +1,48 @@
+import { Controller, Get, Req, Res } from '@nestjs/common';
+
+import { Request, Response } from 'express';
+
+import {
+  AvailableProducts,
+  BillingService,
+  PriceData,
+} from 'src/core/billing/billing.service';
+import { StripeService } from 'src/core/billing/stripe.service';
+
+@Controller('stripe/product-prices/*')
```

Comment:
Question: what is the goal of this enpdpoint? To be used in the FE, if yes, I would call it billing and do not show stripe witch is an implementation detail